app/coffeescript/components/ui/clusters.coffee

```diff
@@ -5,25 +5,23 @@ define [
   'flight/lib/component',
   'marker-clusterer',
   'map/components/utils/map_utils',
-  'map/components/utils/mobile_detection'
+  'map/components/utils/mobile_detection',
+  'map/components/utils/cluster_opts'
 ], (
   compose,
   defineComponent,
   markerClusterer,
   map_utils,
-  mobileDetection
+  mobileDetection,
+  clusterOpts
 ) ->
 
   initMarkerClusters = ->
```

Comment:
Originally the story was to remove ag deps from the file, but being that there wasn't much to be removed in this particular file @mmaiza and I discussed cleaning up some of the opts being set in the file into a mixin. I did it as a component but will be refactoring to a mixin before updating the PR